src/interface.jl

```diff
@@ -38,26 +39,32 @@ end
 ######################
 
 "Abstract type for a `Bijector`."
-abstract type Bijector end
+abstract type Bijector{N} end
+
+dimension(b::Bijector{N}) where {N} = N
 
 Broadcast.broadcastable(b::Bijector) = Ref(b)
 
 """
 Abstract type for a `Bijector` making use of auto-differentation (AD) to
 implement `jacobian` and, by impliciation, `logabsdetjac`.
 """
-abstract type ADBijector{AD} <: Bijector end
+abstract type ADBijector{AD, N} <: Bijector{N} end
 
 """
     inv(b::Bijector)
     Inversed(b::Bijector)
 
 A `Bijector` representing the inverse transform of `b`.
 """
-struct Inversed{B <: Bijector} <: Bijector
+# TODO: can we do something like `Bijector{N}` instead?
```

Comment:
What does this comment mean?

Reply:
Since `Inversed{<: Bijector{D}, D}` is specifying the dimensionality `D` _twice_, I was hoping that there would be some way of dropping the second `D`. Don't think this can be done though, and the current approach is probably as good as it's going to get.

Reply:
I guess you can do
```julia
struct Inversed{N} <: Bijector{N}
    orig::Bijector{N}
end
```
if you want. But it's not type stable right?